common/src/main/java/org/apache/celeborn/common/network/server/TransportServer.java

```diff
@@ -130,24 +131,24 @@ protected void initChannel(SocketChannel ch) {
 
   @Override
   public void close() {
-    shutdown(true);
+    shutdown(CelebornExitStatus.WORKER_GRACEFUL_SHUTDOWN());
   }
 
-  public void shutdown(boolean graceful) {
+  public void shutdown(int exitCode) {
```

Comment:
`exitCode` is definitely misleading, it usually represents the process exit code. Use `exitKind` or simply `kind` instead.

Reply:
Done